include/corosync/totem/totem.h

```diff
@@ -151,6 +151,11 @@ struct memb_ring_id {
 	unsigned long long seq;
 } __attribute__((packed));
 
+enum cfg_message_reload_phase {
+	RELOAD_PHASE_ACTIVATE = 1,
```

Comment:
I'm wondering if it would make sense to have same terminology - there is reload` but functions are called `reconfigure`.